<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCad_Telefone</class>
 <widget class="QDialog" name="frmCad_Telefone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fone</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/Arquivos/Imagens/telefone.png</normaloff>:/Arquivos/Imagens/telefone.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="dDILabel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>DDI :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="dDILineEdit">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>55</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="dDDLabel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>DDD :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="dDDLineEdit">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>062</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="numeroLabel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Numero :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="numeroLineEdit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
     <property name="icon">
      <iconset resource="arquivos.qrc">
       <normaloff>:/Arquivos/Imagens/cancel.png</normaloff>:/Arquivos/Imagens/cancel.png</iconset>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="arquivos.qrc">
       <normaloff>:/Arquivos/Imagens/OK.png</normaloff>:/Arquivos/Imagens/OK.png</iconset>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numeroLineEdit</tabstop>
  <tabstop>dDDLineEdit</tabstop>
  <tabstop>dDILineEdit</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
